sql/CMakeLists.txt

```diff
@@ -1098,136 +1098,7 @@ ENDIF()
 OPTION(DEBUG_EXTNAME "Build server as mysqld-debug (debug builds only)" OFF)
 MARK_AS_ADVANCED(DEBUG_EXTNAME)
 
-<<<<<<< HEAD
 INCLUDE(${CMAKE_SOURCE_DIR}/cmake/bison.cmake)
-=======
-IF(DEBUG_EXTNAME)
-  SET_TARGET_PROPERTIES(mysqld PROPERTIES DEBUG_OUTPUT_NAME "mysqld-debug")
-ENDIF()
-
-# macOS has problems with enabling LTO for shared libs, so we only set it
-# as target property for mysqld.
-IF(APPLE AND WITH_LTO)
-  SET_TARGET_PROPERTIES(mysqld PROPERTIES INTERPROCEDURAL_OPTIMIZATION TRUE)
-ENDIF()
-
-IF(APPLE)
-  # Add CoreServices framework since some dloadable plugins may need it
-  FIND_LIBRARY(CORESERVICES NAMES CoreServices)
-  IF(CORESERVICES)
-    TARGET_LINK_LIBRARIES(mysqld ${CORESERVICES})
-  ENDIF()
```

Comment:
Ibd2sdi is present in runtime_output_directory but during installation we exclude it.  The behavior is the same in 8.0.29 and 8.0.30. I am trying to find out how we do that. Will update

Reply:
@rahulmalik87 may be there is SKIP_INSTALL. Please grep for it.

Reply:
thanks @satya-bodapati . so that's how we skip install
`MYSQL_ADD_EXECUTABLE(ibd2sdi
  ibd2sdi.cc
  COMPONENT Server
  DEPENDENCIES GenError
  LINK_LIBRARIES mysys innodb_zipdecompress ${LZ4_LIBRARY}
  SKIP_INSTALL
  )`

Reply:
@rahulmalik87 please verify after install, only xtrabackup,xbcloud,xbstream binaries are present

Reply:
bin contains
`libprotobuf-lite.3.19.4.dylib	xbcloud				xbcrypt				xtrabackup
libprotobuf.3.19.4.dylib	xbcloud_osenv			xbstream`
which is same as PXB 8.0.29